daemon/emer-daemon.c

```diff
@@ -2050,7 +2051,14 @@ emer_daemon_start_aggregate_timer (EmerDaemon       *self,
                           g_object_ref (self),
                           g_object_unref);
 
-  if (g_hash_table_contains (priv->aggregate_timers, timer_impl))
+  timer_hash_string =
+    emer_aggregate_timer_impl_compose_hash_string (sender_name,
+                                                   unix_user_id,
+                                                   event_id,
+                                                   aggregate_key,
+                                                   nullable_payload);
+
+  if (g_hash_table_contains (priv->aggregate_timers, timer_hash_string))
```

Comment:
The error message below can now be adjusted to include this hash string (which also makes it more accurate since the sender_name is now part of the unique-ifying key)

Reply:
Oh wait, of course, this message is no longer present. Nice.